shared/lib-rabbitmq-connector/src/main/java/gov/va/vro/camel/MessageQueueConfiguration.java

```diff
@@ -30,6 +31,10 @@ public class MessageQueueConfiguration {
   private final String BIE_EVENTS_CONTENTION_COMPLETED = "bie-events-contention-completed";
   private final String BIE_EVENTS_CONTENTION_DELETED = "bie-events-contention-deleted";
   private final String SAVE_TO_DB_PREFIX = "saveToDB-";
+  private final String QUEUE_MESSAGES_DLQ = "bipDeadLetterQueue";
```

Comment:
Should BIE contention events be routed to a different dead letter exchange / queue?

Reply:
It should be fine if they share it.

Reply:
Technically, it would be! I'm just considering future use cases of dealing with the dead lettered messages in a more compartmentalized way such as different handling of messages originating from different services.

Reply:
I changed it to 'vroDeadLetterQueue' so it's less confusing to see bie events in it. I think keeping it to one queue and one exchange is best for now because it keeps the config a lot simpler - we'd have to add a config option for each queue's DLX and DLQ. We'd also have more overhead because we're manually creating alarms for each one. I think we should put the one queue into production to start and see how it goes, and if we find that there's a queue that needs special attention we can go from there.